File: app/src/main/java/ar/com/wolox/android/training/ui/login/LoginModule.java
package ar.com.wolox.android.training.ui.login;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class LoginModule {

    @ContributesAndroidInjector
    public abstract LoginActivity loginActivity();

    @ContributesAndroidInjector
    public abstract LoginFragment loginFragment();
}

File: app/src/main/java/ar/com/wolox/android/training/ui/login/LoginModule.kt
package ar.com.wolox.android.training.ui.login

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class LoginModule {

    @ContributesAndroidInjector
    internal abstract fun loginActivity(): LoginActivity

    @ContributesAndroidInjector
    internal abstract fun loginFragment(): LoginFragment
}
